pubsub/api/PubsubSample/Program.cs

```diff
@@ -236,28 +237,26 @@ public static object CreateTopic(string projectId, string topicId)
             return 0;
         }
 
-        public static PublisherClient GetPublisher(string projectId,
+        public static async Task<PublisherClient> GetPublisherAsync(string projectId,
             string topicId)
         {
             // [START pubsub_publish]
-            PublisherServiceApiClient publisherClient = PublisherServiceApiClient.Create();
-            PublisherClient publisher = PublisherClient.Create(
-                new TopicName(projectId, topicId), new[] { publisherClient });
+            PublisherClient publisher = await PublisherClient.CreateAsync(
+                new TopicName(projectId, topicId));
             // [END pubsub_publish]
             return publisher;
         }
 
         /// <summary>
         /// Create a PublisherClient with custom batch thresholds.
         /// </summary>
-        public static PublisherClient GetCustomPublisher(string projectId,
-            string topicId)
+        public static async Task<PublisherClient> GetCustomPublisherAsync(

```

Comment:
Similar here and I guess in other places

Reply:
Same story.